<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceEditDialog</class>
 <widget class="QDialog" name="PlaceEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditCoordinates"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Комментарий:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Координаты:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="textEditNote"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Другие названия:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="textEditAnotherNames"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxType">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>деревня</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>село</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>город</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>приход</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>наместничество</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>сельское общество</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>волость</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>уезд</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>губерния</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>область</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>страна</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>округ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Тип:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUnionPlaces">
         <property name="text">
          <string>Объединить с</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRecordSearch">
         <property name="text">
          <string>Упоминания</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>textEditAnotherNames</tabstop>
  <tabstop>lineEditCoordinates</tabstop>
  <tabstop>textEditNote</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
